packages/downloader/download.py

```
@@ -0,0 +1,126 @@
+    "skip_unavailable_fragments": True,
+    "hls_prefer_native": True,
+    "buffersize": 1024,
+    # YouTube specific
+    "youtube_include_dash_manifest": True,
+    "prefer_insecure": True,
+    "keepvideo": False,
+    "extract_flat": False,
+    "extractor_args": {
+        "youtube": {
+            "player_client": ["web"],
+        }
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)